packages/algoliasearch-helper/index.d.ts

```diff
@@ -1457,6 +1463,25 @@ declare namespace algoliasearchHelper {
       exhaustive: boolean;
     }
   }
+
+  export type PlainRecommendParameters =
+    | RecommendationsQuery
+    | TrendingQuery
+    | RecommendedForYouQuery;
+
+  export type PlainRecommendParametersWithId = PlainRecommendParameters & {
+    $$id: string;
+  };
+
+  export class RecommendParameters {
+    params: PlainRecommendParametersWithId[];
+    constructor(newParameters?: PlainRecommendParametersWithId);
+    addParams(params: PlainRecommendParametersWithId): RecommendParameters;
+    removeParams(id: string): RecommendParameters;
+    static make(
```

Comment:
do we need this or is it just for symmetry with SearchParameters? As there isn't much reason for SearchParameters.make to exist (it's basically a copy of the constructor with a little extra code for hierarchical menu)